test/spec/import/BpmnTreeWalkerSpec.js

```diff
@@ -50,6 +51,35 @@ describe('import - BpmnTreeWalker', function() {
   });
 
 
+  it('should always call element visitor with parent', function() {
+
+    // given
+    var elementSpy = sinon.spy(),
+        errorSpy = sinon.spy();
+
+
+    var walker = createWalker({
+      element: elementSpy,
+      root: function() {
+        return 'root';
+      },
+      error: errorSpy
+    });
+
+    return createModdle(collaboration).then(function(result) {
+
+      var definitions = result.rootElement;
+
+      // when
+      walker.handleDefinitions(definitions);
+
+      // then
+      expect(elementSpy.neverCalledWith(sinon.match.any, sinon.match.typeOf('undefined'))).to.be.true;
+      expect(errorSpy.notCalled).to.be.true;

```

Comment:
```suggestion
      expect(errorSpy).not.to.be.called;

```

Reply:
:+1: 

Will change it with the other tests in this file as well